src/views/templates/utils/virtualMachineTemplatesCPUMemory.ts

```diff
@@ -0,0 +1,42 @@
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { V1Template } from '@kubevirt-utils/models';
+import { getFlavorData } from '@kubevirt-utils/resources/template';
+
+enum BinaryUnit {
+  B = 'B',
+  Ki = 'Ki',
+  Mi = 'Mi',
+  Gi = 'Gi',
+  Ti = 'Ti',
+}
+
+const toIECUnit = (unit: BinaryUnit | string): string => {
+  if (!unit || unit.endsWith('B')) {
+    return unit;
+  }
+
+  return `${unit}B`;
+};
+
+const stringValueUnitSplit = (combinedVal: string): [value: string, unit: string] => {
+  const index = combinedVal.search(/([a-zA-Z]+)/g);
+  let value = '';
+  let unit = '';
+
+  if (index === -1) {
+    value = combinedVal;
+  } else {
+    value = combinedVal.slice(0, index);
+    unit = combinedVal.slice(index);
+  }
+
+  return [value, toIECUnit(unit)];
+};
+
+export const useVirtualMachineTemplatesCPUMemory = (template: V1Template): string => {
```

Comment:
same here, since this is a hook, it will be best if you transfer this into a separate file, under hooks directory IMO